models/fundModel.js

```
@@ -337,63 +366,55 @@ function fundsModel() {
       funds.forEach(entry => {
         const monthYear = entry.date.substring(0, 7); // Extracting "YYYY-MM"
         if (currentYearMonths.includes(monthYear)) {
           monthlyTotals[monthYear] += entry.money;
         }
       });
-  
+
       // Convert object to array and sort by month
       const sortedMonthlyTotals = Object.entries(monthlyTotals)
         .sort(([a], [b]) => a.localeCompare(b))
         .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)